staging/src/k8s.io/cli-runtime/pkg/resource/builder.go

```diff
@@ -710,6 +712,11 @@ func (b *Builder) SingleResourceType() *Builder {
 	return b
 }
 
+func (b *Builder) NoSelector() *Builder {
```

Comment:
Please add a comment here of what is NoSelector and what it does. 

As an example "NoSelector" will cause the builder to error if selector is not required and the user does not specify a user name

Reply:
follow @rikatz advice, add information "NoSelector will cause the builder to error if selector is not required and the user does not specify a resource name" for NoSelector interface.
is't ok? @soltysh